opts/opts.go

```diff
@@ -203,6 +203,9 @@ func ValidatePath(val string) (string, error) {
 	}
 
 	splited := strings.SplitN(val, ":", 2)
+	if splited[0] == "" {
+		return val, fmt.Errorf("bad format for volumes: %s", val)
+	}
 	if len(splited) == 1 {
 		containerPath = splited[0]
 		val = path.Clean(splited[0])
```

Comment:
To be clear, I think the parsing of this is always going to be a bit ambiguous because you can never really tell if `rw:rw` is hostPath:containerPath or containerPath:rw.  I would suggest that when we only have one ":" we look at the 2nd part and if its `rw` or `ro` then we assume the first part is a containerPath.  Anything other than `rw` and `ro` means we have hostPath:containerPath.  Not foolproof, but we have no other way of knowing what the user meant.